include/libopencm3/pac55xx/ccs.h

```diff
@@ -0,0 +1,148 @@
+/** @file
+ *
+ *  Clock Control and System Defines for the Qorvo PAC55xx series of microcontrollers.
+ *
+ *  @defgroup system_defines Clock Config and System Defines.
+ *  @ingroup PAC55xx
+ *  @author Brian Viele <vielster@allocor.tech>
+ *  LGPL License Terms @ref lgpl_license
+ *  @date 1 Dec 2019
+ *
+ *  Definitions in this file come from the PAC55xx Family User Manual which can
+ *  be found here.
+ *  https://active-semi.com/wp-content/uploads/PAC55XX_Family_User_Guide.pdf
```

Comment:
fwiw, a document id and revision are often better than a url, as vendors have wonderful habits of breaking them, especially we _already_ know that active-semi doesn'te xist anymore.